tests/foreman/cli/test_ansible.py

```diff
@@ -186,6 +191,61 @@ def test_add_and_remove_ansible_role_hostgroup(self, target_sat):
         )
         assert 'Ansible role has been disassociated.' in result[0]['message']
 
+    @pytest.mark.tier3
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    def test_positive_ansible_variables_installed_with_collection(
+        self, request, target_sat, module_org, module_ak_with_cv, rhel_contenthost
+    ):
+        """Verify that installing an Ansible collection also imports
+        any variables associated with the roles avaialble in the collection
+
+        :id: 7ff88022-fe9b-482f-a6bb-3922036a1e1c
+
+        :steps:
```

Comment:
There is no mention of installing the collection here.

Reply:
@nofaralfasi ah, I missed it earlier, updated. Thank you :)